### PR TITLE
octoprint: 1.3.10 -> 1.3.11

### PR DESCRIPTION
**Pull request by gebner**
###### Motivation for this change

The cura slicer plugin is now no longer bundled, you need to add it manually to `services.octoprint.plugins`.

Tested on aarch64, prints fine.

cc maintainer @abbradar 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/octoprint/plugins.nix [hunk 1]
```diff
@@ -104,6 +104,25 @@ let
       };
     };
 
+    curaenginelegacy = buildPlugin rec {
```
Comment (abbradar) [inline pkgs/applications/misc/octoprint/plugins.nix]: Hm, if it's named `legacy` I guess they finally implemented a plugin for new `curaengine` too.

Comment (gebner) [reply pkgs/applications/misc/octoprint/plugins.nix]: Unfortunately no, as far as I can tell.  They just decided to get rid of the ancient cura version.

**Comment by abbradar** [comment]
Ah, I still had `nox-review` running. There is some breakage due to `markdown` update but seems nothing serious.

**Comment by gebner** [comment]
Oops, forgot about that.  What packages are broken?

**Comment by abbradar** [comment]
I'll say when I get home again :D I remember `MarkdownSuperscript`.

**Comment by gebner** [comment]
Ok, `MarkdownSuperscript` is now fixed in https://github.com/NixOS/nixpkgs/commit/879a5c8d8f70a63700c18d9b4d458a54250086ca  Luckily, nothing depends on it.

**Comment by gebner** [comment]
Nikola and pelican are also fixed now.

**Pull request merged**